src/docs/metis.mdx

```diff
@@ -0,0 +1,163 @@
+
+    <Parameter name="Transaction Types" tooltip="The types of transactions supported on the Rollup" />
+
+        - **Type 0** - A compressed EIP155 transaction. The transaction that is signed follows EIP155. The type enum is `0`.
+        - **Type 1** - A compressed EIP155 transaction that uses an alternative signature hashing algorithm. The data is ABI encoded hashed and then signed with `eth_sign`. The type enum is `1`.
+    ___
+
+    <MultiRowParameters title="Messaging" tooltip="The Rollup provides native communication with L1. Anyone is able to transmit arbitrary messages and the Metis protocol will provide proof of the data transfer on the destination chain" data={[
+        {
+            title: 'L1 → L2',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: 'The time it takes for a message to be made available on the Rollup after it is included in the Ethereum block.',
+                    value: '~3 minutes',
+                },
+                {
+                    label: 'Cost',
+                    tooltip: 'The gas cost of purely sending a message from Ethereum to Metis, excluding the actual execution that is scheduled to take place on the L2. Senders on L1 pay for the execution of the message on L2. The payment is equivalent to a normal L2 transaction.',
+                    value: '~170 000 L1 gas',
+                },
+            ],
+        },
+        {
+            title: 'L2 → L1',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: 'The time it takes for a message to be made available on Ethereum after being included in a Metis block and sequenced to L1.',
+                    value: '7 days',
+                },
+                {
+                    label: 'Cost',
+                    tooltip: 'The gas cost of proving and executing a message on Ethereum after being sent from Metis',
+                    value: '740 000 L1 gas',
+                },
+            ],
+        },
+    ]} />
+
+</Section>
+
+<Section title="OPCODEs">
+
+    > ⚠️ if you're using solidity 0.8.9 or above, the default evm version is London, and it's not supported by Metis, you should set evm version to berlin.
+
+    <Legend />
+
+    The following table encapsulates EVM OPCODE differences of the Rollup compared to the canonical Ethereum L1 implementation.
+
+    | Opcode | Name | Solidity Equivalent | Rollup Behaviour | Ethereum L1 Behaviour |
+    | :----- | :--- | :-------------------| :--------------- | :-------------------- |
+    | 3A | GASPRICE | `blockhash.gasPrice` | returns constant `0` | Get price of gas in current environment <Modified /> |
+    | 40 | BLOCKHASH | `blockhash(block.number)` | returns constant `0x0000000000000000000000000000000000000000000000000000000000000000` | Get the hash of one of the 256 most recent complete blocks <Unsupported /> |
```

Comment:
I think that we should label the `blockhash` opcode as `modified`. If it was unsupported the opcode should be invalid and tx should revert, however I would assume that they do not revert, but rather return the constant (modified) value